hack/generate-yamls.sh

```diff
@@ -94,14 +93,17 @@ ko resolve ${KO_FLAGS} -f config/brokers/mt-channel-broker/ | "${LABEL_YAML_CMD[
 # Create in memory channel yaml
 ko resolve ${KO_FLAGS} -f config/channels/in-memory-channel/ | "${LABEL_YAML_CMD[@]}" > "${EVENTING_IN_MEMORY_CHANNEL_YAML}"
 
-
 all_yamls=(${EVENTING_CORE_YAML} ${EVENTING_CRDS_YAML} ${EVENTING_SUGAR_CONTROLLER_YAML} ${EVENTING_MT_CHANNEL_BROKER_YAML} ${EVENTING_IN_MEMORY_CHANNEL_YAML} ${EVENTING_YAML})
 
-  # # Template for POST_INSTALL usage:
-  # # Create vX.Y.Z post-install job yaml.
-  # ko resolve ${KO_FLAGS} -f config/post-install/vX.Y.Z/ | "${LABEL_YAML_CMD[@]}" > "${POST_INSTALL_YAML}"
-  # # If used, add  ${EVENTING_POST_INSTALL_YAML} to all_yamls,
-  # all_yamls+=(${EVENTING_POST_INSTALL_YAML})
+if [ -d "${YAML_REPO_ROOT}/config/post-install" ]; then
+
+  readonly EVENTING_POST_INSTALL_YAML=${YAML_OUTPUT_DIR}/"eventing-post-install.yaml"
+
+  echo "Resolving post install manifests"
+
+  ko resolve ${KO_FLAGS} -f config/post-install/ | "${LABEL_YAML_CMD[@]}" > "${EVENTING_POST_INSTALL_YAML}"
+  all_yamls+=(${EVENTING_POST_INSTALL_YAML})
+fi
```

Comment:
This will not use our previous naming styple `vX.Y.Z`, which makes sense to me but might have some implication of the release tool. So I would like to have someone familiar with the release process to take a look. @n3wscott

Reply:
@lionelvillard is familiar with the release tool and I tested it before submitting the PR, so we're good to go. :) 
(A dir `vx.y.z` in a VCS doesn't make any sense to me)

Reply:
I have to say I totally agree with you! :)